### ISSUE TITLE
[WARN] PersistentConnection: pc_0 - Auth token revoked: expired_token (Auth token is expired.)

### ISSUE THREAD

**Issue by pejas** [firebase/quickstart-java#6]
My application logs this line from time to time with log level WARN. As described in #5 this is expected behaviour, application recovers and works fine. 

Please change log level for this message to DEBUG (or at least INFO). 

WARN alerts admins and they involve devs to investigate which is unnecessary. 

If created this issue in wrong project please point me to right one.

Best Regards

**Comment by samtstern** [comment]
@mikelehen can you comment on why this is `WARN` and not `DEBUG` or `INFO`.  Seems like a reasonable concern.

**Comment by mikelehen** [comment]
Yeah.  I believe a fix for this is in the works, but not sure when it'll be released.  cc/ @schmidt-sebastian

**Comment by tannerhallman** [comment]
Still having this issue. Any updates on the fix?

**Comment by mikelehen** [comment]
Hrm.  I thought this got fixed.  What version of which SDK (firebase or firebase-admin) are you using?

**Comment by mikelehen** [comment]
Er, woops.  Since this is the quickstart-java, I'll assume it's the java server sdk.

**Comment by tannerhallman** [comment]
Yes it is the java server sdk.

```
<dependency>
			<groupId>com.google.firebase</groupId>
			<artifactId>firebase-server-sdk</artifactId>
			<version>3.0.1</version>
		</dependency>
```

**Comment by mikelehen** [comment]
Yep, that's exactly what you need.  It looks like this quickstart is out of date!  Glad to hear the new SDK solved the problem.

**Comment by samtstern** [comment]
Fixed by #8
